pkg/v1/daemon/write.go

```diff
@@ -44,6 +44,36 @@ var GetImageLoader = func() (ImageLoader, error) {
 	return cli, nil
 }
 
+// WriteAndTag writes the image with the first tag and add the others tags with the
+// client.ImageTag function in order to save unnecesary subsequents Writes
+func WriteAndTag(tags []name.Tag, img v1.Image) (string, error) {
```

Comment:
I think I'd rather see this as a simpler:
```go
func Tag(src, dst name.Tag) error {
}
```

And move this logic of `Write` + `Tag` up into pkg/ko/publish/daemon.go, since it has a similar structure.

Reply:
done.